External/FEXCore/Source/Interface/Core/Dispatcher/Dispatcher.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include <FEXCore/Core/CPUBackend.h>
+#include <FEXCore/Core/SignalDelegator.h>
+
+#include "Interface/Context/Context.h"
+
+#include <stack>
+
+namespace FEXCore::CPU {
+
+struct DispatcherConfig {
+  bool Interpeter = false;
```

Comment:
Can we rename this to something like `InterpreterBackend`?
Immediately it confused me between Interpreter CPU core or Linux Interpreter

Reply:
Renamed to `ExecuteBlockWithCall` which better describes what this config actually does.